.github/workflows/auto_update_price_and_context_window.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   auto_update_price_and_context_window:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744  # v3
```

Comment:
please can we see the sha source for all these. I'd like to validate the shas are valid